etcdctl/ctlv3/command/ep_command.go

```diff
@@ -124,14 +125,34 @@ func epHealthCommandFunc(cmd *cobra.Command, args []string) {
 			// endpoint is health.
 			ctx, cancel := commandCtx(cmd)
 			_, err = cli.Get(ctx, "health")
-			cancel()
 			eh := epHealth{Ep: ep, Health: false, Took: time.Since(st).String()}
 			// permission denied is OK since proposal goes through consensus to get it
 			if err == nil || err == rpctypes.ErrPermissionDenied {
 				eh.Health = true
 			} else {
 				eh.Error = err.Error()
 			}
+
+			if eh.Health == true {
+				resp, err := cli.AlarmList(ctx)
+				if err == nil {
```

Comment:
Break out on the not error case. Save one indentation to make code cleaner.

Reply:
@xiang90 Have reduced one indentation level and committed (after squashing again)